.rubocop_todo.yml

```diff
@@ -372,13 +372,6 @@ Style/CommentedKeyword:
 Style/DoubleNegation:
   Enabled: false
 
-# Offense count: 152
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle.
-# SupportedStyles: compact, expanded
```

Comment:
`compact` is the default, which would lead to 152 corrections, `expanded` seems to be way more preferred in the current codebase, leading to only 24 corrections.